katsdpimager/preprocess.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, image_parameters, grid_parameters, buffer_size):
     def num_channels(self):
         return len(self.image_parameters)
 
-    def _emit(self, elements):
+    def _emit(self, channel, elements):
         """Write an array of compressed elements with the same channel and w
```

Comment:
This docstring sounds like it has to change...

Reply:
What in particular?

Reply:
The previous version had an array of elements which hopefully have the same channel stored internally.

The new version also has an array of elements with the same channel, but I guess there is no way to verify that it is true. And the associated channel is passed in as parameter instead of lifted from the elements itself.

So the current docstring is still true, but I wondered if it needed more embellishment. Happy to keep as is too.